File: src/main/java/top/zbeboy/isy/web/vo/graduate/design/adjustech/GraduationDesignTutorUpdateVo.java
package top.zbeboy.isy.web.vo.graduate.design.adjustech;

import lombok.Data;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by zbeboy on 2017/5/23.
 */
@Data
public class GraduationDesignTutorUpdateVo {
    @NotNull
    @Size(max = 64)
    private String graduationDesignReleaseId;
    @NotNull
    private String graduationDesignTutorId;
    @NotNull
    @Size(max = 64)
    private String graduationDesignTeacherId;
    @NotNull
    private int saveType;
}

File: src/main/java/top/zbeboy/isy/web/vo/graduate/design/adjustech/GraduationDesignTutorUpdateVo.kt
package top.zbeboy.isy.web.vo.graduate.design.adjustech

import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by zbeboy 2018-01-20 .
 **/
open class GraduationDesignTutorUpdateVo {
    @NotNull
    @Size(max = 64)
    var graduationDesignReleaseId: String? = null
    @NotNull
    var graduationDesignTutorId: String? = null
    @NotNull
    @Size(max = 64)
    var graduationDesignTeacherId: String? = null
}